packages/shared/src/hooks/useMutateFilters.ts

```diff
@@ -389,25 +389,23 @@ export default function useMutateFilters(
     [user, queryClient, feedId],
   );
 
-  const { mutateAsync: followSourceRemote } = useMutation<
+  const { mutateAsync: unblockSourceRemote } = useMutation<
```

Comment:
In my humble opinion, I think we should not have refactored these ones, it made a lot of avoidable changes.